src/views/forms/FormExtendNamespaceDurationTransaction/FormExtendNamespaceDurationTransaction.vue

```diff
@@ -53,17 +53,17 @@
               :immediate="true"
               slim
             >
-              <input v-show="false" v-model="newDuration" @change="validate">
-              <ErrorTooltip :errors="errors">
+              <!-- <input v-show="false" v-model="newDuration" @change="validate"> -->
```

Comment:
why removing this input? it is breaking the namespace extension form.

Reply:
I commit the code by accident